doc/sphinx/Pacemaker_Explained/resources.rst

```diff
@@ -263,38 +263,52 @@ where to find that resource agent and what standards it conforms to.
+   |             |                                                               |
+   |             | Your name for the resource                                    |
+   +-------------+---------------------------------------------------------------+
+   | class       | .. index::                                                    |
+   |             |    single: class; resource                                    |
+   |             |    single: resource; property, class                          |
+   |             |                                                               |
+   |             | The standard the resource agent conforms to. Allowed values:  |
+   |             | ``lsb``, ``nagios``, ``ocf``, ``service``, ``stonith``,       |
+   |             | ``systemd``, ``upstart``                                      |
+   +-------------+---------------------------------------------------------------+
+   | description | .. index::                                                    |
+   |             |    single: type; resource                                     |
+   |             |    single: resource; property, type                           |
+   |             |                                                               |
+   |             | A description of the Resource Agent, intended for local use.  |
+   |             | E.g. ``IP address for website``                               |
+   +-------------+---------------------------------------------------------------+
+   | type        | .. index::                                                    |
+   |             |    single: type; resource                                     |
+   |             |    single: resource; property, type                           |
+   |             |                                                               |
+   |             | The name of the Resource Agent you wish to use. E.g.          |
+   |             | ``IPaddr`` or ``Filesystem``                                  |
+   +-------------+---------------------------------------------------------------+
+   | provider    | .. index::                                                    |
+   |             |    single: provider; resource                                 |
+   |             |    single: resource; property, provider                       |
+   |             |                                                               |
+   |             | The OCF spec allows multiple vendors to supply the same resource |
+   |             | agent. To use the OCF resource agents supplied by the Heartbeat  |
+   |             | project, you would specify ``heartbeat`` here.                |
+   +-------------+---------------------------------------------------------------+
+
+A resource "description" can be set and retreived with:
+
+.. code-block:: none
+
+   # cibadmin --modify --xml-text '<primitive id="RESOURCE_NAME" description="DESCRIPTION"/>'
+   # crm_resource --list
```

Comment:
We don't need to document the command lines here, the new table entry is sufficient. We do need to document it for groups, clones, and bundles as well.